test/parallel/test-util.js

```diff
@@ -148,6 +148,8 @@ assert.strictEqual(util.isFunction(function() {}), true);
 assert.strictEqual(util.isFunction(), false);
 assert.strictEqual(util.isFunction('string'), false);
 
+assert.strictEqual(util.toUSVString('string'), 'string');
```

Comment:
Can you add a test with an actual unpaired surrogate?

Reply:
Tbh. I don’t know how to make such a string.

Reply:
`'\ud801'` for example